extensions/spring-xd-extension-jdbc/src/main/java/org/springframework/xd/jdbc/NamedColumnJdbcItemReaderFactory.java

```diff
@@ -59,6 +59,8 @@
 
 	private NamedColumnJdbcItemReader reader;
 
+	private String delimiter;
```

Comment:
I think the way it is handled is correct. We might want to enclose string values in quotes to avoid issues with values containing the delimiter character. Then we have the problem with strings containing new line and tab characters. I guess that would be a different JIRA though since that would be new functionality.